### PR TITLE
Fix some errors of tip 33

### PR DESCRIPTION
**Pull request by sjfkai**
## TL;DR;

[content here]
## Username

[@sjfkai](https://github.com/sjfkai)
## Extra

[content here]

### CODE REVIEW

**Comment by zenopopovici** [comment]
Thanks @sjfkai ;)

File: _posts/en/2016-02-02-create-range-0...n-easily-using-one-line.md [hunk 1]
```diff
@@ -27,22 +30,30 @@ Array.apply(null, {length: N}).map(Function.call, Number);
...
 ```js
 Array.from(new Array(N),(val,index)=>index);
 ```
+
 #### Brief explanation
+
 1. `A = new Array(N)` returns an array with `N` _holes_ (i.e. `A = [,,,...]`, but `A[x] = undefined` for `x` in `0...N-1`).
 2. `F = (val,index)=>index` is simply `function F (val, index) { return index; }`
-3. `Array.from(A, F)` returns an `N`-element array, whose index `I` gets the results of `F(A[I], I) == I`.
+3. `Array.from(A, F)` returns an `N`-element array, whose index `I` gets the results of `F(A[I], I) => I`.
```
Comment (gromgit) [inline _posts/en/2016-02-02-create-range-0...n-easily-using-one-line.md:42]: In hindsight, `==` is probably confusing, but `=>` suggests "fat arrow" syntax, which is also confusing.

Perhaps `F(A[I], I) --> I` would be best.

Comment (loverajoel) [inline _posts/en/2016-02-02-create-range-0...n-easily-using-one-line.md:42]: +1

Comment (gromgit) [inline _posts/en/2016-02-02-create-range-0...n-easily-using-one-line.md:42]: Or perhaps:

`Array.from(A, F)` returns an `N`-element array, whose index `I` gets the results of `F(A[I], I)`, which is simply `I`.

**Comment by sjfkai** [comment]
@gromgit  @loverajoel  You are right. I will update this directly.

**Pull request merged**